psi4/src/psi4/cc/ccenergy/get_params.cc

```diff
@@ -165,7 +165,6 @@ void CCEnergyWavefunction::get_params(Options &options) {
 
     params_.print_mp2_amps = options.get_bool("MP2_AMPS_PRINT");
     params_.print_pair_energies = options.get_bool("PAIR_ENERGIES_PRINT");
-    params_.spinadapt_energies = options.get_bool("SPINADAPT_ENERGIES");
```

Comment:
This looks like the only place in the code where `SPINADAPT_ENERGIES` is used, so you can also remove it from `read_options.cc`